model-optimizer/extensions/ops/elementwise.py

```diff
@@ -205,9 +206,17 @@ class Minimum(Elementwise):
 class Round(Elementwise):
     enabled = False
     op = 'Round'
-    op_type = None
-    version = 'extension'
+    op_type = 'Round'
+    version = 'opset4'
     operation = staticmethod(lambda a: np.round(a))
+    def __init__(self, graph: Graph, attrs):
+        elu_attrs = {'in_ports_count': 1}
+        elu_attrs.update(attrs)
+        super().__init__(graph, elu_attrs)
```

Comment:
Add default value for attribute `mode`. Refer to my comment in the specification PR.

Reply:
Done, added 'mode' attribute with default value 'half_to_even'